requirements.txt

```diff
@@ -4,3 +4,4 @@ pyyaml
 dargs >= 0.2.2
 tqdm
 typing_extensions
+sphinx_markdown_tables
```

Comment:
Move this package to the following line, as we do not need it unless building the document.
https://github.com/deepmodeling/deepmd-kit/blob/40c6ae2a70b8aa6eb2c3e64dd928d63efa7e4083/setup.py#L110

Reply:
ok